packages/kbn-securitysolution-exceptions-common/scripts/openapi_bundle.js

```diff
@@ -33,7 +33,17 @@ const ROOT = resolve(__dirname, '..');
             name: 'Security Exceptions API',
             'x-displayName': 'Security exceptions',
             description:
-              "Exceptions API allows you to manage detection rule exceptions to prevent a rule from generating an alert from incoming events even when the rule's other criteria are met.",
+              "Exceptions are associated with detection and endpoint rules, and are used to prevent a rule from generating an alert from incoming events, even when the rule's other criteria are met. They can help reduce the number of false positives and prevent trusted processes and network activity from generating unnecessary alerts.\n\n\
+Exceptions are made up of:\n\n\
+* **Exception containers**: A container for related exceptions. Generally, a single exception container contains all the exception items relevant for a subset of rules. For example, a container can be used to group together network-related exceptions that are relevant for a large number of network rules. The container can then be associated with all the relevant rules.\n\
+* **Exception items**: The query (fields, values, and logic) used to prevent rules from generating alerts. When an exception item's query evaluates to `true`, the rule does not generate an alert.\n\n\
+For detection rules, you can also use lists to define rule exceptions. A list holds multiple values of the same Elasticsearch data type, such as IP addresses. These values are used to determine when an exception prevents an alert from being generated.\n\
+> info\n\
+> You cannot use lists with endpoint rule exceptions.\n\n\
+> info\n\
+> Only exception containers can be associated with rules. You cannot directly associate an exception item or a list container with a rule. To use list exceptions, create an exception item that references the relevant list container.\n\n\
+## Exceptions requirements\n\n\
+Before you can start working with exceptions that use value lists, you must create the `.lists` and `.items` data streams for the relevant Kibana space. To do this, use the Create list data streams endpoint. Once these data streams are created, your role needs privileges to manage rules. For a complete list of requirements, refer to [Enable and access detections](https://www.elastic.co/guide/en/serverless/current/security-detections-requirements.html#enable-detections-ui).",
```

Comment:
We use docLinks service in Kibana to get references to documentation and make sure links won't get outdated.
Since it's just JS script, it's integration here might not be easy.

cc @maximpn, since you worked on that script